wagtail/admin/templates/wagtailadmin/edit_handlers/tabbed_interface.html

```diff
@@ -1,8 +1,8 @@
 {% load wagtailadmin_tags %}
-<ul class="tab-nav merged" role="tablist">
+<ul class="tab-nav merged" role="tablist" aria-owns="{% for child in self.children %}tab-label-{{ child.heading|cautious_slugify }} {% endfor %}">
     {% for child in self.children %}
-        <li class="{{ child.classes|join:" " }} {% if forloop.first %}active{% endif %}" role="tab" aria-controls="tab-{{ child.heading|cautious_slugify }}">
-            <a href="#tab-{{ child.heading|cautious_slugify }}" class="{% if forloop.first %}active{% endif %}">{{ child.heading }}</a>
+        <li role="presentation" {% if child.classes %}class="{{ child.classes|join:" " }}"{% endif %}>
```

Comment:
I thought `role="presentation"` would prevent content within this from being displayed?

Reply:
Not in the testing I did with VoiceOver and Safari. I made this choice after using Firefox's accessibility inspector and seeing these results. Without `role="presentation"` on the `li`s (notice the three `nothing` items):

<img width="1436" alt="Screen Shot 2019-07-27 at 8 25 00 PM" src="https://user-images.githubusercontent.com/296631/62000745-c061a900-b0ac-11e9-82a6-63333493243d.png">

With `role="presentation"` on the `li`s:
  
<img width="1438" alt="Screen Shot 2019-07-27 at 8 24 24 PM" src="https://user-images.githubusercontent.com/296631/62000747-c9eb1100-b0ac-11e9-9a52-61117d26f932.png">

As far as I can tell though, testing with VoiceOver, there's no behavioral difference I observed.

Reply:
This seems like the relevant documentation. It's a little confusing to me, but it seems to be saying descendant elements only inherit the `presentation` role in special cases that don't apply here https://www.w3.org/TR/wai-aria-practices-1.1/#presentation_role_effects

Happy to defer to your judgment though on whether it's preferable to leave in, take out, of restructure the HTML and CSS 😵

Reply:
Ah I see, I didn’t realise this wasn’t inherited.